tests/examples/custom/custom.py

```diff
@@ -14,6 +14,8 @@ def __init__(self, env, name):
 
     def run(self):
         self.logger.info('Running test case {}'.format(self.name))
-        vm = self.env.domains[0]
-        [retcode, output] = vm.execute_cmd('ip address show')
+        vm1 = self.env.domains[0]
+        vm2 = self.env.domains[1]
+        vm1.execute_cmd('ip address show')
+        vm2.execute_cmd('ip address show')
```

Comment:
Only Question: Do we have the network address of a domain already in some variable?

Reply:
Currently, it is stored in the domain object as a single variable "ip" and it's retrieved from the terraform output command. You pointed another thing to improve!